### PR TITLE
optimized typings installation

### PR DESCRIPTION
**Pull request by sclausen**
To remove the hen egg problem with typings I migrated the tsd commands from the gulp-task to the beginning of the npm postinstall script, right before gulp executes.

### CODE REVIEW

**Comment by ludohenin** [comment]
@sclausen would you please rebase (we have some conflict since I merge alpha42)?

File: tools/tasks/tsd.ts [hunk 1]
```diff
@@ -1,7 +0,0 @@
-export = function tsd(gulp, plugins) {
```
Comment (tarlepp) [inline tools/tasks/tsd.ts]: I think that this task is still usefull in some of cases, so I would not remove this.

**Comment by sclausen** [comment]
I might have messed up the commit history a little…

**Comment by mgechev** [comment]
No worries, if you prefer would you open a PR in a fresh branch?

**Comment by sclausen** [comment]
I hope this one is cleaner #173

**Pull request closed without merging**